index.html

```diff
@@ -6,44 +6,54 @@
     <meta name="Author" content="Manizha khorram">
     <link rel="stylesheet" type="text/css" href="Style.css" />
 </head>
-<header>
-    <div id="div4nav">
-    </div>
-    <nav>
-        <a href="#About/">About</a> 
-        <a href="#Experience/" id="experience">Experience</a> 
-        <a href="#Connect/" id="connect">Connect</a> 
-    </nav>
-
-    <h1 id="myName">Manizha khorram</h1>
-
-    <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="myImage">
-
-</header>
 
-<body>
-
-    <section class="myInfo">
-        <h2><a href="#About">About</a></h2>
+<body class="containerAll">
+    <header>
+        <div id="div4nav">
+        </div>
+        <nav>
+            <a href="#About">About</a>
+            <a href="#Experience" >Experience</a>
+            <a href="#skills" >Skills</a>
+        </nav>
+    
+        <h1 id="myName">Manizha khorram</h1>
+    
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="myImage">
+    
+    </header>
+    
+    <main>
+    <section class="myInfo" id="About">
+        <h2 >About</h2>
         <p id="about">I love coding. I would like to work as professional sofware developer. I hold bachelor of computer
             science degree, I love problem solving and creating new things.I keen to learn challenging roles in
             javaScript, software development and related technologies.</p>
     </section>
 
-    <section class="myInfo">
-        <h2><a href="#Experience">Experience</a></h2>
-        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University
-            Work Force Development Program) in Afghanistan.</p>
+    <section class="myInfo" id="Experience">
+        <h2>Experience</h2>
+        <div id="container">
+        <p>System development analyst-NPA (National Procurement Authority)-Afghanistan</p>
+        <p>06/2018 - 04/2019</p>
+        <p> System development analyst-USWDP (University Work Force Development Program)-Afghanistan.</p>
+        <p>02/2018 - 06/2018</p>
+    </div>
     </section>
 
     <section id="skills" class="myInfo">
-        <h2><a href="#Connect">Skills</a></h2>
+        <h2>Skills</h2>
+        <ul></ul>
+
+    </section>
+    <section id="projects" class="myInfo">
+        <h2>Projects</h2>
         <ul></ul>
```

Comment:
here I have the projects section and also (ul), and locally for myself it is working, what do you think would be the reason? is something wrong with github?!